ontology/yaml/resources/HVAC/entity_types/ABSTRACT.yaml

```diff
@@ -6618,3 +6621,28 @@ VMADC:
   - mixed_air_damper_percentage_sensor
   implements:
   - CONTROL
+
+GWH3X:
```

Comment:
Make a combination with the `HWTC` type you have below to try to form something similar to `HT4SWC`